modules/claims_api/spec/requests/v2/veterans/rswag_power_of_attorney_spec.rb

```diff
@@ -169,7 +169,7 @@
         }
       end
 
-      describe 'Getting a successful response' do
+      xdescribe 'Getting a successful response', document: false do
```

Comment:
pending the specs that were brought over for endpoint we won't support. We might want to reference and/or delete them later.